monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+
+#### Integraciones
+Colaborar para poder integrar a Monero en una variedad de servicios, esto incluye procesadores de pagos, casas de cambio descentralizadas y más.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/serhack-monero-integrations](https://taiga.getmonero.org/project/serhack-monero-integrations/) & #monero-integrations
+
+#### Labratorio de transmisiones de Monero
+Apuntamos a resolver los problemas relacionados a la transmisión de las transacciones de Monero a través una infraestructura descentralizada de código abierto (redes mesh, satélites, etc.)
+_¿Dónde empezar?_ [taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/108](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/108)
+
+### _Únete a la conversación e Involucrate_
+---
+- Lista de correos - [lists.getmonero.org/postorius/lists/monero-announce.lists.getmonero.org](https://www.monerooutreach.org/stories/lists.getmonero.org/postorius/lists/monero-announce.lists.getmonero.org)
+- Reddit - [reddit.com/r/Monero](https://www.reddit.com/r/Monero/)
+- Twitter - [twitter.com/monero](https://twitter.com/monero)
+- Facebook - [facebook.com/monerocurrency](https://facebook.com/monerocurrency)
+- Taiga - [taiga.getmonero.org/discover/search?order_by=-total_fans](https://taiga.getmonero.org/discover/search?order_by=-total_fans)
+- Stack Exchange - [monero.stackexchange.com](https://monero.stackexchange.com/)
+- Quora - [quora.com/topic/Monero](https://quora.com/topic/Monero)
+- IRC - [web.getmonero.org/community/hangouts](https://web.getmonero.org/community/hangouts) o busca  “monero” en IRC para la lista completa. Muchas de estás tienen puentes a Mattermost, Slack, Matrix, etc.
+- Canales de Telegram – [t.me/monerotelegram](https://t.me/monerotelegram)
+
+### _Otras maneras de ayudar_
+---
+- Ejecuta un nodo completo - [monerooutreach.org/monero_best_practices/your_own_node.html](https://www.monerooutreach.org/monero_best_practices/your_own_node.html) y [monero.how/how-to-run-monero-node](https://www.monero.how/how-to-run-monero-node)
+- Dona al CCS (Sistema de Financiación Colectiva de la Comunidad) - [ccs.getmonero.org](https://ccs.getmonero.org/)
+- Minar Monero - [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+- Únete a algún equipo de trabajo - [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+- Contribuir a futuras versiones del libro Mastering Monero (Dominando a Monero) - [github.com/monerobook/monerobook](https://github.com/monerobook/monerobook) y [masteringmonero.com](https://masteringmonero.com/)
+- Responder preguntas en Stack Exchange - [monero.stackexchange.com](https://monero.stackexchange.com/)
+- Conviertete en un Mercaderista de guerrilla - [monerooutreach.org/guerrilla-toolkit.html](https://www.monerooutreach.org/guerrilla-toolkit.html)
```

Comment:
To follow verbs:

_Contribuye_.
_Responde_.

And this is just missing:

_Conviértete_

Reply:
Fixed.